src/common/number/format_number.ts

```diff
@@ -77,6 +77,24 @@ export const formatNumber = (
       ).format(Number(num));
     }
   }
+
+  if (
+    !Number.isNaN(Number(num)) &&
+    num !== "" &&
+    localeOptions?.number_format === NumberFormat.none &&
+    Intl
```

Comment:
```suggestion
    Intl && (options?.maximumFractionDigits || options?.minimumFractionDigits)
```
Should we only do this when there is a precision specified?